test/meteor-test.js

```diff
@@ -36,18 +36,12 @@ describe('Meteor', function() {
       assert.isBelow(this.meteor.size.height, 66);
     });
 
-    it('should have a height and width that can be manually set', function() {
```

Comment:
Since you're removing this test, it might be good to write a new test or update your existing tests to explicity state the new functionality. IE 'it should determine dimension based on level'